Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fuel-merkle"
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
Nit, but can you bump version in a separate PR? We don't necessarily want to tag that version right away.